resources/Makefile.diff

```diff
@@ -0,0 +1,36 @@
+3,6c3,6
+< obj-$(CONFIG_LOGO)			+= logo.o
+< obj-$(CONFIG_LOGO_LINUX_MONO)		+= logo_linux_mono.o
+< obj-$(CONFIG_LOGO_LINUX_VGA16)		+= logo_linux_vga16.o
+< obj-$(CONFIG_LOGO_LINUX_CLUT224)	+= logo_linux_clut224.o
+---
+> obj-$(CONFIG_LOGO)									+= logo.o
+> obj-$(CONFIG_LOGO_LINUX_MONO)				+= logo_linux_mono.o
+> obj-$(CONFIG_LOGO_LINUX_VGA16)			+= logo_linux_vga16.o
+> obj-$(CONFIG_LOGO_LINUX_CLUT224)		+= logo_linux_clut224.o
+8,19c8,19
+< obj-$(CONFIG_LOGO_BLACKFIN_VGA16)	+= logo_blackfin_vga16.o
+< obj-$(CONFIG_LOGO_DEC_CLUT224)		+= logo_dec_clut224.o
+< obj-$(CONFIG_LOGO_MAC_CLUT224)		+= logo_mac_clut224.o
+< obj-$(CONFIG_LOGO_PARISC_CLUT224)	+= logo_parisc_clut224.o
+< obj-$(CONFIG_LOGO_SGI_CLUT224)		+= logo_sgi_clut224.o
+< obj-$(CONFIG_LOGO_SUN_CLUT224)		+= logo_sun_clut224.o
+< obj-$(CONFIG_LOGO_SUPERH_MONO)		+= logo_superh_mono.o
+< obj-$(CONFIG_LOGO_SUPERH_VGA16)		+= logo_superh_vga16.o
+< obj-$(CONFIG_LOGO_SUPERH_CLUT224)	+= logo_superh_clut224.o
+< obj-$(CONFIG_LOGO_M32R_CLUT224)		+= logo_m32r_clut224.o
+< 
+< obj-$(CONFIG_SPU_BASE)			+= logo_spe_clut224.o
+---
+> obj-$(CONFIG_LOGO_BLACKFIN_VGA16)		+= logo_blackfin_vga16.o
+> obj-$(CONFIG_LOGO_DEC_CLUT224)			+= logo_dec_clut224.o
+> obj-$(CONFIG_LOGO_MAC_CLUT224)			+= logo_mac_clut224.o
+> obj-$(CONFIG_LOGO_PARISC_CLUT224)		+= logo_parisc_clut224.o
+> obj-$(CONFIG_LOGO_SGI_CLUT224)			+= logo_sgi_clut224.o
+> obj-$(CONFIG_LOGO_SUN_CLUT224)			+= logo_sun_clut224.o
+> obj-$(CONFIG_LOGO_SUPERH_MONO)			+= logo_superh_mono.o
+> obj-$(CONFIG_LOGO_SUPERH_VGA16)			+= logo_superh_vga16.o
+> obj-$(CONFIG_LOGO_SUPERH_CLUT224)		+= logo_superh_clut224.o
+> obj-$(CONFIG_LOGO_M32R_CLUT224)			+= logo_m32r_clut224.o
+> obj-$(CONFIG_LOGO_NODESOS_CLUT224)	+= logo_nodeos_clut224.o
```

Comment:
Just do the patch with this line and/or the needed ones, not the full file. Are you doing the patches by hand? I've found several typos, like `NODESOS` instead of `NODEOS`... Also, what dialect are you using? I haven't seen before a patch defining the changes with `<` and `>`, but instead with `+` and `+`...